### ISSUE TITLE
Suggestion: Examples are lacking best practices

### ISSUE THREAD

**Issue by albvar** [lazywinadmin/WinFormPS#3]
The function help examples for the majority of the examples are really hard to follow.
The function help examples should leverage examples that the consumers can easily plug in and visualize the output without having to understand the dependencies from function to function. A good way I like to think of this is as follows. Put yourself in the shoes of someone who has never seen this before, how can they quickly figure out if this module does what they need.

**Comment by albvar** [comment]
Also you are using Aliases incorrectly. Example, you have Alias "Load-DataGridView" and "Refresh-DataGridview" aliases are not supposed to be another verb-noun convention for an existing function. Instead they are meant to call commands using shorthand, such as gci -> Get-ChildItem,  ogv -> Out-GridView, leverage Get-Alias and you can quickly see how to properly implemented aliases per best practices.

**Comment by lazywinadmin** [comment]
@albvar Thanks for the feedback! Appreciated. This is a old project of mine as you can see and I learned a lot since. I'll try to update it eventually.

Feel free to submit a PR
